en/guides/using-open-source-software.md

```diff
@@ -1,176 +1,101 @@
-In these cases, this investment must be considered by weighing the total cost of ownership against those of other candidate solutions.
-
-### Evaluation
-
-The same factors applicable to an evaluation of the feature set and maturity of closed-source software also apply to open source software.
-Additional criteria should be assessed when evaluating open source software:
+### Evaluating Open Source Software
+The same factors used to evaluate the feature set and maturity of closed-source software, apply to open source software. These additional criteria are specific to the evaluation of open source software:
 
 #### User Community
-
-A strong user community involved in a project provides people to answer questions, test the software, report bugs, suggest improvements and drive forward overall interest in the software.
-Look at the software's public code repository and check the project popularity by looking at the number of likes and followers.
-Check how active the project responds to users by looking at issues and the time between replies.
+A strong user community involved in a project provides people to answer questions, test the software, report bugs, suggest improvements and drive forward overall interest in the software. The engagement of the user community can be assessed by the open source software project’s social media activity, mentions, likes and followers. 
 
 #### Developer Community
+Open source software is created and maintained by its developer community. This community corresponds to the manufacturer of proprietary software. A strong developer community with a history of releases and continued involvement is an indicator of longevity. 
 
-A strong developer community with a history of releases and continued involvement tends to demonstrate that fixes and improvements to the software will continue into the future.
-Look at who are the core developers and who is supporting the project and community, such as a non-profit Foundation.
-Look at when the project got started, the pace of releases and responses to requests to merge code from contributors.
+Positive indicators:
+* Project supported by a non-profit foundation 
+* Developer community composed of leaders within the industry or knowledge domain.
+* Quick pace of releases, and responses to requests to merge code from contributors.
 
 #### Documentation
-
 User documentation provides important information to help users install software and use its features. Technical documentation provides requirements and instructions for installation, development, deployment and configuration of the software.
 
-#### Security Assessments
-
+#### Security Management
 Although OSS code is auditable, this does not necessarily mean it is secure. The quality of the code and the typical response time for patching security-related flaws help indicate the security level of the software.
+Note: The project should have a process to list all packages and maintain versions.
```

Comment:
This was more a TODO on the GC part: we need to have a thorough tracking of packages (including FOSS as COTS) since we don't monitor "licences" like proprietary software.